packages/@o3r/localization/README.md

```diff
@@ -23,702 +33,16 @@ ng add @o3r/localization
-
-**Lets assume:**
-
-```typescript
-{
-  "supportedLocales": ['en-GB', 'en-US', 'fr-FR', 'ar-AR'],
-  "fallbackLocalesMap": {
-            'en-CA': 'en-US',
-            'fr-CA': 'fr-FR',
-            'de-CH': 'ar-AR',
-            'de': 'fr-FR',
-            'it': 'fr-FR',
-            'hi': 'en-GB',
-            'zh': 'en-GB'
-  },
-  "fallbackLanguage": 'ar-AR'
-}
-```
-
-**Fallback scenario's:**
-
-en-CA **fallbacks to** en-US, as direct mapping available in fallback Locale Map.
-
-de-CH **fallbacks to** ar-AR, as direct mapping available in fallback Locale Map.
-
-de-AT **fallbacks to** fr-FR, as language mapping available in fallback Locale Map.
-
-zh-CN **fallbacks to** en-GB, as language mapping available in fallback Locale Map.
-
-en-AU **fallbacks to** en-GB, as fallback locales mapping unavailable, first nearest language available in supported locales.
-
-fr-BE **fallbacks to** fr-FR, as fallback locales mapping unavailable, first nearest language available in supported locales.
-
-bn-BD **fallbacks to** ar-AR, as it is the default fallback.
-
-### Scenario 2: Fallback based on **supportedLocales** language code
-
-In case if **fallbackLocalesMap** is not provided and the targeted translation language is unavailable is supported locales list, The targeted translation language will be matched with supported locales **language** to find out the first nearest match, the **locale** can be different. If none match, it will fallback to the **default language**.
-
-**Lets assume:**
-
-```typescript
-{
-  "supportedLocales": ['en-GB', 'fr-FR', 'fr-CA', 'ar-AR'],
-  "fallbackLanguage": 'en-GB'
-}
-```
-
-**Fallback scenario's:**
-
-en-US **fallbacks to** en-GB, as en-GB has the same language with a different region.
-
-fr-BE **fallbacks to** fr-FR, as fr-FR is first in the supported locales list.
+## Generators
 
-it-IT **fallbacks to** en-GB, as it is the default fallback.
+Otter framework provides a set of code generators based on [Angular schematics](https://angular.io/guide/schematics).
 
-## Generators
+| Schematics                    | Description                                                 | How to use                           |
+|-------------------------------|-------------------------------------------------------------|--------------------------------------|
+| add                           | Include Otter localization module in a library/application. | `ng add @o3r/localization`           |
+| localization-to-component     | Add localization architecture to an Otter component         | `ng g localization-to-component`     |
+| localization-key-to-component | Add a localization key to an Otter component                | `ng g localization-key-to-component` |
 
-Otter framework provides a set of code generators based on [angular schematics](https://angular.io/guide/schematics).
+## More details
```

Comment:
Maybe we can add that with the vscode extension you can localize easily from the template file

Reply:
👍 I will create an issue for this once the PR is merged